lessons/introduction-to-smartpy/05/05.mdx

```diff
@@ -6,17 +6,21 @@ editor:
   language: python
   startingCode: |
     import smartpy as sp
+
     class Plant(sp.Contract):
         def __init__(self, name):
-        		self.init(name = sp.string(name))
-    
+
+            # Initialize state variables
+        		self.init(name = sp.string(name)) 
+
   answer: |  
     import smartpy as sp
+
     class Plant(sp.Contract):
         def __init__(self, name):
-        		self.init(name = sp.string(name),
-            growth_rate = sp.int(10), health = sp.int(100))
-    
+
+            # Initialize state variables
+        		self.init(name = sp.string(name), attack = sp.int(10), defense = sp.int(10), growth_rate = sp.int(10), health = sp.int(100))
```

Comment:
beak line so that user can see the code without horizontal scroll, this ruins the UX

Reply:
If we break the line, it will show error by the compiler.